### ISSUE TITLE
publicPath relative

### ISSUE THREAD

**Issue by ManuelMariggio** [shuidi-fed/vapper#74]
Hello,

I'm trying to build the application with relative path,

using the **publicPath** param in **vue.config.js** in this way
```js
module.exports = {
    publicPath: './'
}
```

 so in the index.html i expect:

```js
<script src=js/runtime~app.5bf8a270.js></script>
<script src=js/chunk-vendors.8fcd5f97.js></script>
<script src=js/app.d8974dd5.js></script>
```

but the build produce
```js
<script src=/_vapper_/js/runtime~app.3964bcd2.js></script>
<script src=/_vapper_/js/chunk-vendors.8fcd5f97.js></script>
<script src=/_vapper_/js/app.875dd5a5.js></script>
```

How can i avoid the /vapper folder  writing?

i have also tried this configuration but without success

```js
module.exports = {
    publicPath: ''
}
```
```js
module.exports = {
    publicPath: process.env.BASE_URL,
    assetsDir: process.env.BASE_URL,
}
```
Can you help me?
Thank you

**Comment by HcySunYang** [comment]
Which version are you using? Can you provide a reproducible example?

**Comment by ManuelMariggio** [comment]
Hello,

i have posted the example in a repository here: https://github.com/ManuelMariggio/simplevapperbuild

Thank you in advance

**Comment by HcySunYang** [comment]
Thanks for your feedback, we have fixed it, please use `v1.8.0`.

**Comment by ManuelMariggio** [comment]
Hello,

it works perfectly using 

```js
module.exports = {
    publicPath: ''
}
```

Thank you!
Best regards
